src/HttpKernel/TestKernel.php

```diff
@@ -34,8 +34,10 @@
      * concrete kernel configure itself using the abstracvt
      * configure() command.
      */
-    public function init()
+    public function __construct($env, $debug)
     {
+        parent::__construct($env, $debug);
```

Comment:
I think we have to move this to the bottom of the overriden construct method. Otherwise, bundle sets won't be registered when the init method is executed.